packages/fints/src/__tests__/test-pin-tan-client-e2e.ts

```diff
@@ -0,0 +1,76 @@
+import * as fs from "fs";
+import { TanRequiredError } from "../errors/tan-required-error";
+import { PinTanClient } from "../pin-tan-client";
+import { SEPAAccount } from "../types";
+
+const url = process.env.FINTS_URL;
+const name = process.env.FINTS_USER;
+const pin = process.env.FINTS_PASSWORD;
+const blz = process.env.FINTS_BLZ;
+
+// const url = process.env.COMDIRECT_FINTS_URL;
+// const name = process.env.COMDIRECT_FINTS_USER;
+// const pin = process.env.COMDIRECT_FINTS_PASSWORD;
+// const blz = process.env.COMDIRECT_FINTS_BLZ;
+
+// const url = process.env.ING_DIBA_FINTS_URL;
+// const name = process.env.ING_DIBA_FINTS_USER;
+// const pin = process.env.ING_DIBA_FINTS_PASSWORD;
+// const blz = process.env.ING_DIBA_FINTS_BLZ;
+
+const productId = "9FA6681DEC0CF3046BFC2F8A6";
+
+test.skip("get accounts", async () => {
```

Comment:
Either delete these tests or the `.skip`.

Reply:
Maybe I must move this 'test' to the fints-cli package. I used it during development.

Reply:
I'd suggest to create a new type of tests (perhaps in a different sub-package?) called E2E tests. We could move this kind of tests in there and only execute them on demand. What do you think?

Reply:
That's a good idea, too. But I noticed that the fints-cli is broken with the new fints module. I want to repair this too. And this e2e-test is indeed a fints-cli executions. Having said that, I think this test should be removed.

Reply:
OK